inbm/dispatcher-agent/dispatcher/validators.py

```diff
@@ -17,14 +17,19 @@
 logger = logging.getLogger(__name__)
 
 
-def get_schema_location(schema_type: str, schema_location: Optional[str] = None) -> str:
+def _get_schema_location(schema_location: Optional[str] = None) -> str:
     if not schema_location:
         schema_location = JSON_SCHEMA_LOCATION
     return schema_location
 
+"""Validates JSON against a JSON schema
 
-def validate_schema(schema_type: str, params: str, schema_location: Optional[str] = None) -> str:
-    schema_location = get_schema_location(schema_type, schema_location)
+@param params: JSON Parameters
+@param schema_location: JSON schema location.  Default=NONE
+@return: Deserialized JSON
+"""
+def _validate_schema(params: str, schema_location: Optional[str] = None) -> str:
```

Comment:
if returning deserialized JSON, type annotation of returning `str` is incorrect

Reply:
That was there.  I just added the comments and made it private.  But I'll change it to any.